README.md

```diff
@@ -40,9 +38,13 @@ Action is a Node.js application based upon the
 | Data Transport     | [GraphQL](https://github.com/graphql/graphql-js)          |
 | Sockets            | [socketcluster](http://socketcluster.io/)                 |
 | Client State       | [Redux](http://redux.js.org/)                             |
-| Front-end          | [React](https://facebook.github.io/react/)                |
+| Client Data Cache  | [Cashay](https://github.com/mattkrick/cashay)             |
+| Front-end Views    | [React](https://facebook.github.io/react/)                |
 | Styling            | [react-look](https://github.com/rofrischmann/react-look/) |
 
+Action is programmed in ECMAscript ES6/7 (including async/await).
```

Comment:
Not sure if this is worth calling out anymore? ES2015 is universally adopted, await/async is officially part of the spec & should be included in V8 within a month or 2.